packages/mrm-task-readme/templates/Readme.md

```diff
@@ -5,11 +5,7 @@
 ## Changelog
 
 The changelog can be found on the [Releases page](${github}/releases).
-
-## Contributing
-
-Everyone is welcome to contribute. Please take a moment to review the [contributing guidelines](Contributing.md).
-
+${contributing}
```

Comment:
It might be cleaner to keep all the new lines in the template and then remove all double empty lines from the resulting text.

```suggestion

${contributing}
```

Reply:
If you mean so that all users get double empty lines removed, besides that I'm not sure all users would want this, this text is built with the `mrm-core` template utility which would I guess need to allow a new method to set the content directly (so it can be saved properly to file by `save()`. Just want to be sure if that's what you want.

If you mean just for the sake of the tests, I can do that, but then it might confuse users who might look at the template and think it should behave the same.

Let me know which way you want to go--thanks!

Reply:
Yeah, that might be tricky to implement.

Would something like this works?

```suggestion
${contributing && '\n' + contributing}
```

Reply:
Sure, that works, pushed (along with rebasing).